docs/resources/mail_server.md

```diff
@@ -0,0 +1,57 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "artifactory_mail_server Resource - terraform-provider-artifactory"
+subcategory: "Configuration"
+---
+
+# Artifactory Mail Server Resource
+
+Provides an Artifactory Mail Server resource. This can be used to create and manage Artifactory mail server configuration.
+
+## Example Usages
+
+```terraform
+resource "artifactory_mail_server" "mymailserver" {
+    enabled         = true
+    artifactory_url = "http://tempurl.org"
+    from            = "test@jfrog.com"
+    host            = "http://tempurl.org"
+    username        = "test-user"
+    password        = "test-password"
+    port            = 25
+    subject_prefix  = "[Test]"
+    use_ssl         = true
+    use_tls         = true
+}
+```
+
+## Argument reference
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `enabled` (Boolean) When set, mail notifications are enabled.
+- `host` (String) The mail server IP address / DNS.
+- `port` (Number) The port number of the mail server.
+
+### Optional
+
+- `artifactory_url` (String) The Artifactory URL to to link to in all outgoing messages.
+- `from` (String) The 'from' address header to use in all outgoing messages.
+- `password` (String) The password for authentication with the mail server.
+- `subject_prefix` (String) A prefix to use for the subject of all outgoing mails.
+- `use_ssl` (Boolean) When set to 'true', uses a secure connection to the mail server.
```

Comment:
mmm, can these both be true? tls and ssl?

Reply:
Yes